source/guides/dashboard/introduction.md

```diff
@@ -87,3 +87,10 @@ Once you log in to the {% url 'Dashboard Service' https://on.cypress.io/dashboar
 - [{% fa fa-folder-open-o %} cypress-example-kitchensink](https://dashboard.cypress.io/#/projects/4b7344)
 - [{% fa fa-folder-open-o %} cypress-example-todomvc](https://dashboard.cypress.io/#/projects/245obj)
 - [{% fa fa-folder-open-o %} cypress-example-piechopper](https://dashboard.cypress.io/#/projects/fuduzp)
+
+# Next Steps
+
+<!-- Line breaks removed to prevent random br elements -->
+{% note info %}
+<p class="section-title">Test and debug faster with the Cypress Dashboard</p><ul class="mb-2"><li>See exact point of failure of tests running in CI</li><li>Supercharge test times by running tests in parallel</li><li>Get instant test failure alerts via Slack or GitHub</li></ul><a href="https://www.cypress.io/dashboard" class="button">Get Started Free</a>
```

Comment:
Not really seeing why this needs to be HTML other than the anchor link since we don't support classes in url tags (although we could). 

Suggestion without HTML looks about the same, if not cleaner without the extra spacing on the top and bottom of the note block.

```suggestion
### Test and debug faster with the Cypress Dashboard

- See exact point of failure of tests running in CI
- Supercharge test times by running tests in parallel
- Get instant test failure alerts via Slack or GitHub

<a href="https://www.cypress.io/dashboard" class="button">Get Started Free</a>
```

### Current

<img width="892" alt="Screen Shot 2020-07-16 at 1 52 50 PM" src="https://user-images.githubusercontent.com/1271364/87639715-ab71a000-c76b-11ea-859e-bc68845dee77.png">


### Suggestion

<img width="885" alt="Screen Shot 2020-07-16 at 1 49 26 PM" src="https://user-images.githubusercontent.com/1271364/87639497-5b92d900-c76b-11ea-8c6b-aadd2e310b1d.png">

Reply:
Weird. I was having issue with Hexo adding a bunch of `<br>` tags into the snippet which is why I had to keep collapsing them and turning into tags. Looks like it's fine now? Thanks for the suggestion!